MKDataScanner/MKDataScannerDataProvider.m

```diff
@@ -32,7 +32,8 @@ - (NSData *)dataForRange:(NSRange)range
 
 - (BOOL)isAtEnd
 {
-    return (self.offset < self.data.length);
+//    return (self.offset < self.data.length);
+    return (self.offset >= self.data.length);
```

Comment:
Hi Marcin, cool, let me know if I need to do anything,Ferga
      From: Marcin Krzyzanowski <notifications@github.com>
 To: krzyzanowskim/MKDataScanner <MKDataScanner@noreply.github.com> 
Cc: FergalMohan <fergalmohan@yahoo.com> 
 Sent: Monday, October 12, 2015 9:09 PM
 Subject: Re: [MKDataScanner] Please consider this Pull Request (#3)
   
In MKDataScanner/MKDataScannerDataProvider.m:> @@ -32,7 +32,8 @@ - (NSData *)dataForRange:(NSRange)range
>  
>  - (BOOL)isAtEnd
>  {
> -    return (self.offset < self.data.length);
> +//    return (self.offset < self.data.length);
> +    return (self.offset >= self.data.length);
 👍
no need for this commented code though.—
Reply to this email directly or view it on GitHub.